src/options/normalizeSQL.js

```diff
@@ -12,10 +12,16 @@ function normalizeDate (where) {
   return where
 }
 
+/**
+ * Transform the input of requested response fields
+ * @param {Object} options - object that may contain 'fields' or 'outFields' property
+ */
 function normalizeFields (options) {
   const fields = options.fields || options.outFields
   if (fields === '*') return undefined
-  return typeof fields === 'string' ? [fields] : fields
+  if (typeof fields === 'string' || fields instanceof String) return fields.split(',')
```

Comment:
What's a scenario where `fields` would not be type `string` but also an instance of `String`?  Is that some JS weirdness that can happen with prototypes?

Reply:
It's probably overkill here, but that can happen when a string is created with `new String('Some text')`, which has type Object.  Check it out `console.log(typeof new String("test"))`.  So I usually default to string checking like this.

Reply:
Oh, Javascript, is there nothing you can't make confusing?